ReactAndroid/build.gradle

```diff
@@ -293,30 +293,30 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'src/main/third-party/java/infer-annotations/', include: ['*.jar'])
-    compile 'javax.inject:javax.inject:1'
-    compile 'com.android.support:appcompat-v7:27.1.1'
-    compile 'com.facebook.fbui.textlayoutbuilder:textlayoutbuilder:1.0.0'
-    compile "com.facebook.fresco:fresco:${FRESCO_VERSION}"
-    compile "com.facebook.fresco:imagepipeline-okhttp3:${FRESCO_VERSION}"
-    compile 'com.facebook.soloader:soloader:0.5.1'
-    compile 'com.google.code.findbugs:jsr305:3.0.2'
-    compile "com.squareup.okhttp3:okhttp:${OKHTTP_VERSION}"
-    compile "com.squareup.okhttp3:okhttp-urlconnection:${OKHTTP_VERSION}"
-    compile 'com.squareup.okio:okio:1.14.0'
+    api fileTree(dir: 'src/main/third-party/java/infer-annotations/', include: ['*.jar'])
```

Comment:
Why use `implementation` for some gradle files and `api` for others?

Reply:
test framework don't need to expose.